src/test_suite/multiprocessing_utils.py

```diff
@@ -173,25 +207,39 @@ def decode_single_test_case(test_file: Path) -> int:
     Returns:
         - int: 1 if successfully decoded and written, 0 if skipped.
     """
-    serialized_instruction_context = read_context_serialized(test_file)
+    if test_file.suffix == ".fix":
+        fn_entrypoint = extract_metadata(test_file).fn_entrypoint
+        harness_ctx = HARNESS_ENTRYPOINT_MAP[fn_entrypoint]
+        serialized_protobuf = read_fixture_serialized(test_file)
```

Comment:
Would `test_file.read_text()` work? Why do we need a separate function?

Reply:
later